docker/files/daemon.json

```diff
@@ -1,5 +1,8 @@
 {% from "docker/map.jinja" import host with context %}
 {
+  {%- if host.experimental is defined %}
+  "experimental": host.experimental ,
```

Comment:
If host.insecure_registries is undefined, comma at the end of this line will cause invalid json.